pkgs/tools/misc/brltty/default.nix

```diff
@@ -46,7 +49,10 @@ stdenv.mkDerivation rec {
     "--with-writable-directory=/run/brltty"
     "--with-updatable-directory=/var/lib/brltty"
     "--with-api-socket-path=/var/lib/BrlAPI"
-  ];
+  ] ++ lib.optional espeak-ngSupport [
+      "--with-espeak_ng=$espeak-ng"
+      "--with-speech-driver=en"
```

Comment:
Just wondering if we might want to use speechd. That way, user could choose other speech drivers without having to rebuild this.